packages/sitecore-jss/src/i18n/rest.dictionary-service.ts

```diff
@@ -0,0 +1,76 @@
+import { AxiosDataFetcher } from '../axios-fetcher';
+import { fetchData } from '../data-api';
+import { HttpDataFetcher } from '../data-fetcher';
+import { DictionaryPhrases, DictionaryServiceBase, CacheOptions } from './dictionary-service';
+
+/**
+ * A reply from the REST Sitecore Dictionary Service
+ */
+export type RestDictionaryServiceData = {
```

Comment:
@anastasiya29 You didn't export it from the package. But if somebody will implement custom RestDictionaryService and wants to use data types? instead of duplicate it

Reply:
This came out of a conversation I had with Nick about whether we're exporting too many abstractions from the base packages.

From Nick: "if I want to do something funky with graphql-request or Apollo in my components, don't I want to just look at docs for those and not have to figure out how to translate it through Sitecore's abstraction of that?"

So the question is, does it really add value for devs to have access to all these abstractions, or are we just making the API overly complex? I haven't formed my opinion yet, but I figured it's safe to exclude this particular export since it's never used in the sample app.

Do you think it should be added back?

Reply:
@anastasiya29 Understood. In this case I'm also not sure about it. But let's imagine that I want to do something with graphql-request or Apollo and anyway in this case I need some interfaces from our package.
For me `RestDictionaryServiceData` it's a standard sitecore dictionary response and it can be reused by somebody. But to be honest I'm not sure that somebody will implement custom dictionary service.
Let's suppose that if somebody need to implement custom dictionary service so they will add own data interface.